.github/workflows/iroha2-dev-pr.yml

```diff
@@ -70,18 +76,23 @@ jobs:
             clang \
             llvm-dev
       - name: Run tests
-        run: cargo test --workspace --no-fail-fast -- --test-threads=1
+        run: cargo test --workspace --no-fail-fast -- --test-threads=4
```

Comment:
Why exactly 4?

Reply:
I would wait for network p2p fix. Afterwards we can increase test-threads number.

Reply:
Let's leave it at 1for now then maybe? if we are not certain?

Reply:
We can always change it. It is kind of misc. If it would fail, we will notice.